elements/pfe-tooltip/BaseTooltip.ts

```diff
@@ -20,7 +20,6 @@ export abstract class BaseTooltip extends LitElement {
   abstract position?: Placement;
 
   #float = new FloatingDOMController(this, {
-    arrow: true,
```

Comment:
@brianferry please validate this change

Reply:
@brianferry I was poking at the controller and noticed `arrow` wasn't being used in the code, so I removed it. Let me know if there's something I'm missing.